src/btree/bt_vrfy.c

```diff
@@ -662,6 +663,10 @@ __verify_tree(
              * If configured, continue traversing through the pages of the tree even after
              * encountering errors reading in the page.
              */
+            if (ret != 0 && __wt_ref_addr_copy(session, ref, &addr)) {
+                WT_RET(__wt_msg(session, "ERROR unreadable block at: %s",
+                  __wt_addr_string(session, addr.addr, addr.size, vs->tmp2)));
+            }
```

Comment:
Perhaps we could structure the code from line 666 to 675 a bit differently - 
pseudo code
```
            if (ret != 0) {
                if (dump_address)
                     print failure with the addr string
                if (allowed to read corrupt) {
                    continue;
                } else
                    WT_RET(ret);
            }
```

Reply:
thank you, fixed https://github.com/wiredtiger/wiredtiger/pull/10095/commits/ae539a0f12157ba628ce989ba4883bf7cae3ebe4